frontend/src/app/[locale]/opportunity/[id]/page.tsx

```diff
@@ -124,11 +124,9 @@ async function OpportunityListing({ params }: OpportunityListingProps) {
   });
 
   const nofoPath =
```

Comment:
since we aren't doing anything to try to see if the attachment is a nofo anymore, I think we should remove `nofo` from any variable names, including the props on OpportunityDescription and OpportunityDownload.  Maybe `firstAttachmentPath`?